### PR TITLE
added note in footer in all mailers

### PR DESCRIPTION
**Pull request by ykl7**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue
* [ ] if possible, multiple commits squashed if they're smaller changes
* [x] reviewed/confirmed/tested by another contributor or maintainer
* [x] `schema.rb.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

Refers to #932 #1024 #1068 

@jywarren I've added the outreach note to welcome and password reset mailers as well. Hope that's ok

### CODE REVIEW

File: app/views/answer_mailer/notify_answer_accept.html.erb [hunk 1]
```diff
@@ -5,3 +5,5 @@
 <p>View your answer here: https://<%= ActionMailer::Base.default_url_options[:host] %><%= @answer.node.path(:question) %>#a<%= @answer.id %></p> 
 
 <p>To change your email preferences, please visit https://<%= ActionMailer::Base.default_url_options[:host] %>/subscriptions.</p>
+
+@footer = feature('email-footer')
```
Comment (jywarren) [inline app/views/answer_mailer/notify_answer_accept.html.erb]: Oops, here i think you meant to use `<%= @footer %>`.

File: app/views/answer_mailer/notify_answer_like.html.erb [hunk 2]
```diff
@@ -5,3 +5,5 @@
 <p>View your answer here: https://<%= ActionMailer::Base.default_url_options[:host] %><%= @answer.node.path(:question) %>#a<%= @answer.id %></p> 
 
 <p>To change your email preferences, please visit https://<%= ActionMailer::Base.default_url_options[:host] %>/subscriptions.</p>
+
+@footer = feature('email-footer')
```
Comment (jywarren) [inline app/views/answer_mailer/notify_answer_like.html.erb]: Oops, here i think you meant to use `<%= @footer %>`.

File: app/views/answer_mailer/notify_answer_likers_author.html.erb [hunk 3]
```diff
@@ -16,4 +16,6 @@ Hi! There's been a new answer posted for the question '<a href='https://<%= Acti
 
 <p>Report abuse to: moderators@<%= ActionMailer::Base.default_url_options[:host] %></p>
 
+@footer = feature('email-footer')
```
Comment (jywarren) [inline app/views/answer_mailer/notify_answer_likers_author.html.erb]: Oops, here i think you meant to use `<%= @footer %>`.

File: app/views/answer_mailer/notify_question_author.html.erb [hunk 4]
```diff
@@ -18,4 +18,6 @@ Hi! A new answer has been posted for your question '<a href='https://<%= ActionM
 
 <p>Report abuse to: moderators@<%= ActionMailer::Base.default_url_options[:host] %></p>
 
+@footer = feature('email-footer')
```
Comment (jywarren) [inline app/views/answer_mailer/notify_question_author.html.erb]: Oops, here i think you meant to use `<%= @footer %>`.

**Comment by jywarren** [comment]
Hi, i think there's a small mistake in the actual erb templates -- left a few notes but pls take a look at all of them! Thanks!

**Comment by ykl7** [comment]
@jywarren can't seem to find your notes on the templates. Can you please point me to them? Thanks

**Comment by jywarren** [comment]
Oh, oops, i started a "review" but didn't publish it. Doing so now.

**Comment by ykl7** [comment]
Ah, now I see the errors. Really sorry about those. Getting on fixing all of them

**Comment by jywarren** [comment]
no problem at all -- i'll keep an eye out!

**Comment by ykl7** [comment]
@jywarren Done. Please have a look now

**Comment by jywarren** [comment]
Perfect; merging now. 

Thanks!!! 🎈 🎉

**Pull request merged**